examples/batch.md

```diff
@@ -0,0 +1,29 @@
+### Running batch tasks in xpk
+
+### Preparation
+
+This section should describe how to prepare volumes and add external files executed in script.
+
+### Execution
+
+To run xpk batch script run:
+```bash
+python3 xpk.py batch slurm_sample.sh
```

Comment:
are we allowed to use `slurm`?

Reply:
I think kjob mode we use is named slurm, that is why i named script this way